packages/vite-plugin-svelte/package.json

```diff
@@ -41,23 +41,22 @@
   },
   "homepage": "https://github.com/sveltejs/vite-plugin-svelte#readme",
   "dependencies": {
-    "@sveltejs/vite-plugin-svelte-inspector": "^2.1.0",
+    "@sveltejs/vite-plugin-svelte-inspector": "workspace:^",
```

Comment:
I guess we want to use the static `^3.0.0` after it's published?

Reply:
yes, either that, or inline the inspector into v-p-s entirely, or remove the dependency and let users add it to their vite config manually. The current situation is a bit tedious for maintenance